File: Desktop/tango/see/facedet/usbcam/usbcamCap.cpp
#include "opencv2/highgui/highgui.hpp"

using namespace cv;

int main()
{
  Mat image;
  char key;
  
  VideoCapture cap;
  cap.open(0);
  
  while (cap.isOpened())
  {
    cap>>image;
    
    if (!image.empty())
      imshow("webcam", image);
    
    key = waitKey(33);
    if (char(key) == 27)
    	break;
  }
  
  cap.release();
  destroyAllWindows();
  
  
  return 0;
}

File: Desktop/tango/see/facedet/usbcam/usbcamCap.py
import time
import cv2
    
# allow the camera to warmup
time.sleep(0.1)

# capture frames frome the camera
cap = cv2.VideoCapture(0)

while True:
    ret, frame = cap.read()
    cv2.imshow('faceDetection', frame)
    if cv2.waitKey(10) & 0xFF == ord('q'):
        break
    
cap.release()
cv2.destroyAllWindows()
for x in range(0, 5):
    cv2.waitKey(1)

